sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -79,37 +79,61 @@ export class ServiceBusClient {
   }
 
   /**
-   * Creates a receiver for an Azure Service Bus queue.
+   * Creates a receiver for an Azure Service Bus queue in peekLock mode.
+   *
+   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
+   * queue.
+   *
+   * Messages that are not settled within the lock duration will be redelivered as many times as
+   * the max delivery count set on the queue, after which they get sent to a separate dead letter
+   * queue.
+   *
+   * You can settle a message by calling complete(), abandon(), defer() or deadletter() methods on
+   * the message.
```

Comment:
What do you think of providing a link to https://docs.microsoft.com/en-us/azure/service-bus-messaging/message-transfers-locks-settlement#peeklock at this point? It will help people who wonder what the difference between the 4 methods are instead of us explaining it.

And https://docs.microsoft.com/en-us/azure/service-bus-messaging/message-transfers-locks-settlement#receiveanddelete for the receive and delete mode

Reply:
I'll add in the peekLock link. The receiveAndDelete link isn't adding much value so I'm inclined to leave that one out.

Reply:
Updated!